website/components/footer/index.jsx

```diff
@@ -21,9 +21,7 @@ export default function Footer({ openConsentManager }) {
           <Link href="/security">
             <a>Security</a>
           </Link>
-          <Link href="/files/press-kit.zip">
-            <a>Press Kit</a>
-          </Link>
+          <a href="https://www.hashicorp.com/brand">Press Kit</a>
```

Comment:
I'm not familiar with why the footer wraps `<a>` tags that are links in `<Link>` tags, but just noting that inconsistency. Maybe it should be like this to be consistent?

```suggestion
          <Link href="https://www.hashicorp.com/brand">
            <a>Press Kit</a>
          </Link>
```